tests/integration/helpers/format-money-test.ts

```diff
@@ -7,25 +7,34 @@ module('Integration | Helper | format-money', function (hooks) {
   setupRenderingTest(hooks);
 
   hooks.beforeEach(function () {
-    this.amount = 1234;
+    this.amount = 1234.89;
     this.currency = 'USD';
   });
 
   module('Basic formatting', function () {
     test('Formats money when an amount and a currency are passed', async function (assert) {
       await render(hbs`<div>{{format-money this.amount this.currency}}</div>`);
-      assert.dom('div').hasText('$1,234');
+      assert.dom('div').hasText('$1,234.89');
     });
 
     test('Divides amount by 100 when cents are passed', async function (assert) {
       this.format = 'cents';
 
       await render(hbs`<div>{{format-money this.amount this.currency this.format}}</div>`);
-      assert.dom('div').hasText('$12.34');
+      assert.dom('div').hasText('$12.35');
     });
   });
 
   module('Compact formatting (K/M/B)', function () {
+    test('Before thousands value, it renders complete number when compact is passed as true', async function (assert) {
```

Comment:
nit: but you could import the value directly from the exported const in the helper ?
also `below` instead of `before`

Also I don't see the test for `CENTS`